subgraphs/uniswap-v2/schema.graphql

```diff
@@ -0,0 +1,495 @@
+  outputTokenAmount: BigInt!
+
+  " USD-normalized value of the transaction of the underlying (e.g. sum of tokens deposited into a pool) "
+  amountUSD: BigDecimal!
+
+  ##### DexAmm-Specific #####
+
+  " The pool involving this transaction "
+  pool: LiquidityPool!
+}
+
+type Withdraw implements Event @entity {
+  " { Transaction hash }-{ Log index }"
+  id: ID!
+
+  " Transaction hash of the transaction that emitted this event "
+  hash: String!
+
+  " Event log index. For transactions that don't emit event, create arbitrary index starting from 0 "
+  logIndex: Int!
+  " The protocol this transaction belongs to "
+  protocol: DexAmmProtocol!
+  " Address that received tokens "
+  to: String!
+  " Market that tokens are withdrawn from "
+  from: String!
+  blockNumber: BigInt!
+  timestamp: BigInt!
+  " Input tokens of the transaction. E.g. the UNI-LP token "
+  inputTokens: [Token!]!
+  " Output tokens of the transaction. E.g. WETH and USDC from a WETH-USDC pool "
+  outputTokens: Token!
+  " Amount of input tokens in the token's native unit "
+  inputTokenAmounts: [BigInt!]!
+  " Amount of output tokens in the token's native unit "
+  outputTokenAmount: BigInt!
+  " USD-normalized value of the transaction of the underlying (e.g. sum of tokens withdrawn from a pool) "
+  amountUSD: BigDecimal!
+
+  ##### DexAmm-Specific #####
+
+  " The pool involving this transaction "
+  pool: LiquidityPool!
+}
+
+type Swap implements Event @entity {
+  " { Transaction hash }-{ Log index } "
+  id: ID!
+
+  " Transaction hash of the transaction that emitted this event "
+  hash: String!
+
+  " Event log index. For transactions that don't emit event, create arbitrary index starting from 0 "
+  logIndex: Int!
+  " The protocol this transaction belongs to "
+  protocol: DexAmmProtocol!
+  " Address that received tokens "
+  to: String!
+  " Market that tokens are withdrawn from "
+  from: String!
+  blockNumber: BigInt!
+  timestamp: BigInt!
+  " Token deposited into pool "
+  tokenIn: Token!
+  " Amount of token deposited into pool in native units "
+  amountIn: BigInt!
+  " Amount of token deposited into pool in USD "
+  amountInUSD: BigDecimal!
+  " Token withdrawn from pool "
+  tokenOut: Token!
+  " Amount of token withdrawn from pool in native units "
+  amountOut: BigInt!
+  " Amount of token withdrawn from pool in USD "
+  amountOutUSD: BigDecimal!
+  ##### DexAmm-Specific #####
+  " The pool involving this transaction "
+  pool: LiquidityPool!
+}
+
+#Added Entities 
+
+# An account is a unique Ethereum address
+type _Account @entity {
+  " Address of the account "
+  id: ID!
+}
+
+type _DailyActiveAccount @entity {
+  " { Days since Unix epoch }-{ Address of the account } "
+  id: ID!
+}
+
+type _Transaction @entity {
+  id: ID! # txn hash
+  blockNumber: BigInt!
+  timestamp: BigInt!
+  " This is not the reverse of Mint.transaction; it is only used to track incomplete mints (similar for burns and swaps) "
```

Comment:
Probably should update this comment since we don't have `Mint.transaction` anymore

Reply:
Removed the comment. Fixed as part of the deletion above.